### PR TITLE
Ready for code review

### PR DESCRIPTION
**Pull request by noomdalv**

### CODE REVIEW

**Comment by bolah2009** [comment]
Currently under review

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,866 @@
...
+  }
+
+  .firstGridLeft ul {
+    margin-top: 50px;
+    color: gray;
+  }
+
+  .firstGridLeft h2 {
+    display: flex;
+    flex-direction: wrap;
```
Comment (bolah2009) [inline css/main.css]: `SUGGESTION`

Consider changing to a correct value.

`flex-direction` does not have a `wrap` value. Possible values are `row` `row-reverse`, `column`, `column-reverse` and other global attributes. Learn more about `flex-direction` [here](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,743 @@
...
+
+      <article class="firstSectionRight1">
+        <div class="cover-gradient"></div>
+        <div class="firstGridRight1">
+          <h5>TECH</h5>
+          <h2 class="firstSectionTitle">
+            <a href="https://thenextweb.com/syndication/2019/07/29/hype-is-killing-ai-heres-how-can-we-can-stop-it/">Hype is killing AI – here’s how we can stop it</a>
+          </h2>
+          <ul>
+            <li><a id="firstSectionTitleA" href="https://thenextweb.com/author/ben-dickson/">by Ben Dickson</a></li>
```
Comment (bolah2009) [inline index.html]: `SUGGESTION`

Consider using a unique ID. [See here](https://github.com/noomdalv/Building-with-Responsive-Design/pull/2#discussion_r310845568)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,743 @@
...
+
+      <article class="firstSectionRight2">
+        <div class="cover-gradient"></div>
+        <div class="firstGridRight2">
+          <h5>GEAR</h5>
+          <h2 class="firstSectionTitle">
+            <a href="https://thenextweb.com/syndication/2019/07/29/hype-is-killing-ai-heres-how-can-we-can-stop-it/">Google ad confirms the Pixel 4's face unlock and handwave-y gestures</a>
+          </h2>
+          <ul>
+            <li><a id="firstSectionTitleA" href="https://thenextweb.com/author/napierLopez/">by Napier Lopez</a></li>
```
Comment (bolah2009) [inline index.html]: `SUGGESTION`

You seem to be using this ID more than once, as it appears [here](https://github.com/noomdalv/Building-with-Responsive-Design/pull/2#discussion_r310845420)

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Nice Job Team :clap: 

The clone looks nice, however, I would suggest making it more responsive since the main objective of the project is to make a responsive clone of [this page](https://thenextweb.com/). See below (check the full [page](https://thenextweb.com/) for more section that needs to be responsive:

| Current | Preferred |
| :-----: | :-----: |
| tnw current | tnw preffered |

You can recreate this issue (Image above) by using the console on chrome **(Inspect `Ctrl+Shift+I`)** and the **Toggle device toolbar** `Ctrl+Shift+M`. This way you can simulate how the app will render on small to widescreen devices.

### Optional but strongly recommended

- Note [this comment](#discussion_r310841076) on using `wrap` as `flex-direction` value.

- Check out [these errors](https://validator.w3.org/nu/?doc=https%3A%2F%2Fraw.githack.com%2Fnoomdalv%2FBuilding-with-Responsive-Design%2Ffeature-smallscreenprototype%2Findex.html) too.

Good luck with your next submission.

Cheers
[Bola](https://bolabuari.com/)

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Nicely done team :clap: 
But before I give a :+1: we need to do a few changes to make your clone page behaves just like the original one (these are required by Microverse guidelines). They are:

- Latest news section

- [ ] Must aligns 3 articles per row in the medium version(4 is only for the large version).

- Latest Funding section

- [ ] Must aligns 3 articles per row in the medium version(4 is only for the large version).

- Categories (Apps, GERA, Tech, etc) section

- [ ] Must aligns 2 articles per row in the medium version(3 is only for the large version).

- Latest deals section

- [ ] Must aligns 3 articles per row in the medium version(4 is only for the large version).

Please kindly make these changes and submit for code review again!

**Review by mosaaleb — CHANGES_REQUESTED** [review]
## Well done @noomdalv, @SofiBretz :clap:

I see that you have put a lot of effort into this project. I admire your dedication but there is some requirement needs to be met before moving on.

### Large Screen:
- [ ] Navigation bar contain search icon at the rightmost side -- As seen on the the original page.


### Medium Screen

- [ ] Links are hiding behind the blocks as you can see. let's fix this
- [ ] Also each funding section header text is hiding behind the left thumbnail in All screen sizes.
Screenshot from 2019-08-08 19-57-02

- [ ] Please, use [stickler](https://github.com/microverseinc/linters-config/tree/master/css). It doesn't take time to use and it makes sure that you have good styling guidelines.

- [ ] [OPTIONAL]: I highly recommend the start all of the coming projects using [`reset.css`] (https://meyerweb.com/eric/tools/css/reset/) file (except when you are using bootstrap as it is bundled by default with a reset file) , as you can you see there is some margin at the top of the page, this is default value and can change by browsers. Using `reset.css` file ensure that we have the same initial state in all browsers.


Please handle these requirements and submit a new code review.

### Happy Coding :computer:

**Comment by bolah2009** [comment]
Currently under review

File: css/main.css [hunk 4]
```diff
@@ -0,0 +1,991 @@
...
+    margin-bottom: -50px;
+  }
+
+  #latestDeals {
+    grid-column: 1 / 4;
+  }
+}
+
+@media screen and (max-width: 768px) {
+  #fifthSection {
```
Comment (bolah2009) [inline css/main.css:957]: `HINT`

You seem to be using only two breakpoints for the media query which is affecting the display of this section for smaller screens. See below:

| Current | Preferred |
| :-----: | :-----: |
| tnw current | tnw preffered |

**Comment by Ispirett** [comment]
# Reviewing

**Review by bolah2009 — APPROVED** [review]
## Good Job Team :clap: 

The [clone](https://raw.githack.com/noomdalv/Building-with-Responsive-Design/feature-smallscreenprototype/index.html) looks good to me.

#### Friendly reminder: You should use [Linter](https://github.com/microverseinc/linters-config#linters-config) in this project. Please make sure you use it in the next project. You will find instructions in the Pathwright lesson description.

### Check the comment below before merging.

Good luck with your next project.

Cheers
[Bola](https://bolabuari.com/)

**Comment by noomdalv** [comment]
Hi Bola, thank you for the review!
i just fixed the error and it should be ok now, last commit we were
changing something for the 1024px breakpoint and somehow missed that,
thanks again.
the latest deals section displays the elements in rows now, like this:

[image: image.png]


Bye!.

**Pull request merged**